samples/bookinfo/src/reviews/reviews-wlpcfg/Dockerfile

```diff
@@ -12,14 +12,15 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM websphere-liberty:20.0.0.6-full-java8-ibmjava
+ARG REGISTRY=istio
```

Comment:
I think how this/the script works means we would publish this to `docker.io/istio` which I don't think we want. We only want to publish the bookinfo ones

Reply:
this is needed to make the PR complete/testable, maybe you could update this in a follow up PR with your suggestion https://github.com/istio/istio/pull/37546#discussion_r946202762
 ?

Reply:
if I run this PR though it will publish to docker.io/istio which is what I was trying to avoid. Ill ma ually build the image we need and push it?

Reply:
Building now

Reply:
gcr.io/istio-testing/websphere-liberty:22.0.0.8-full-java8-ibmjava@sha256:2a58e7930d95872741ddef3468e3ec4bba940c577b02d0c48ed6a99b3d6b2dae

Reply:
haven't been able to test the latest changes because the image isn't ready yet
```
docker pull gcr.io/istio-testing/websphere-liberty:22.0.0.8-full-java8-ibmjava
Error response from daemon: manifest for gcr.io/istio-testing/websphere-liberty:22.0.0.8-full-java8-ibmjava not found: manifest unknown: Failed to fetch "22.0.0.8-full-java8-ibmjava" from request "/v2/istio-testing/websphere-liberty/manifests/22.0.0.8-full-java8-ibmjava".
```

Reply:
Oops, published but without the tag. Should be good now